packages/serverless/src/apollo/app/collection-index.ts

```diff
@@ -9,26 +14,25 @@ import {
 } from '../../domain/use-case/collection.js';
 import { gql } from '../../helper/common.js';
 import { authorizeWrapper } from '../validation.js';
+import { CollectionRequest } from './collection.js';
```

Comment:
I think something possible wrong here, all types moved to `@zkdb/common`

Reply:
It's a Joi validation that I re-use between collection and collection-index. The type certainly from @zkdb/common